Sources/Queries/MemberQuery.swift

```diff
@@ -11,7 +11,7 @@ import Foundation
 /**
  A class that facilitates the creation of subqueries against `T`'s custom member properties. Since the creation of this class is initiated in the `PredicateBuilder<T>` class, and this class inherits from it, then member creation is recursive.
  */
-public final class MemberQuery<T: Reflectable, MemberType: Reflectable & AnyObject>: PredicateBuilder<T>, Matchable, NilComparable {
```

Comment:
See the above question about removing Reflectable. The same case goes for the AnyObject constraints I put here.

Reply:
And here ```MemberQuery.swift:14:74: Redundant layout constraint 'MemberType' : 'AnyObject'```